Maintenance/infrastructure/picasso.geometryfactory.com/reference_platforms/x64_Cygwin-Windows8_MSVC2012-Debug-32bits/CMakeCache.txt

```diff
@@ -219,19 +219,20 @@ CMAKE_RC_COMPILER:FILEPATH=C:/Program Files (x86)/Windows Kits/8.0/bin/x86/rc.ex
 CMAKE_RC_FLAGS:STRING=' '
 
 //Flags used by the linker during the creation of dll's.
-CMAKE_SHARED_LINKER_FLAGS:STRING=' /STACK:10000000 /machine:X86 '
+CMAKE_SHARED_LINKER_FLAGS:STRING=' /STACK:10000000 /machine:X86 /DEBUG:NONE /INCREMENTAL:NO'
+CMAKE_STATIC_LINKER_FLAGS:STRING=' /STACK:10000000 /machine:X86 /DEBUG:NONE /INCREMENTAL:NO'
```

Comment:
Arg! That cache is now incorrect. Because `CMAKE_STATIC_LINKER_FLAGS` does not have a description (the empty line followed by the comment line). Try it locally with either `cmake` or `cmake-gui`. Probably CMake will complain.

Is that platform `x64_Cygwin-Windows8_MSVC2012-Debug-32bits` still active?

Reply:
Does seem to be. It is not in recent testsuites.

Reply:
That does not tell use anything, because it might absent from testsuites because of that syntax error.

I had a look, on picasso:
```shellsession
$ grep COMPILER ~/.autocgalrc
COMPILERS_localhost=""
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2012-Release-32bits"
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2012-Release-64bits"
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2012-Debug-64bits"
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2013-Release-32bits"
#COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2013-Release-64bits"
#COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2013-Debug-64bits"
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2013-Debug-32bits"
#COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2010-Debug-32bits"
#COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2010-Release-64bits"
#COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2010-Release-64bits-LEDA"
#COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2008-Debug-64bits"
#COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2008-Release-32bits"
```
... most of the lines are commented, here is the real list:
```shellsession
$ egrep '^COMPILER' ~/.autocgalrc
COMPILERS_localhost=""
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2012-Release-32bits"
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2012-Release-64bits"
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2012-Debug-64bits"
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2013-Release-32bits"
COMPILERS_localhost="$COMPILERS_localhost x64_Cygwin-Windows8_MSVC2013-Debug-32bits"
```

So no: that platform `x64_Cygwin-Windows8_MSVC2012-Debug-32bits` is not supposed to be tested.